Hyper/PlayerData/Camera.cs

```diff
@@ -156,6 +157,30 @@ public void UpdateWithCharacter(Player player)
         }
     }
 
+    public void UpdateWithCar(SimpleCar car)
```

Comment:
I think it would be better to have these in Player/car classes instead of in camera. (by these i mean UpdateWIthCar and UpdateWithCharacter)

Reply:
moving these to SimpleCar would force us to move Camera to Common. I'll move these to controllers